### ISSUE TITLE
Endpoint is not registered when enableRouteMetrics is false

### ISSUE THREAD

**Issue by luiz** [SkeLLLa/fastify-metrics#15]
Reading the documentation, I understood that if pass `enableRouteMetrics: false` at the plugin configuration, the plugin would only stop exporting route metrics, but would still export a route with default metrics if `endpoint` is configured. However, this is not happening. When I use the following configuration, accessing `/metrics` results in 404:

```js
app.register(metricsPlugin, {
	enableRouteMetrics: false,
	endpoint: '/metrics',
});
```

**Comment by SkeLLLa** [comment]
Hi. Want to make a PR that fixes it?

**Comment by luiz** [comment]
Sure! We're working on it.

**Comment by SkeLLLa** [comment]
Released as v6.0.3
